scripts/install-requirements.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Detect the operating system type
+OS="$(uname -s)"
+DISTRO=""
+
+# Define installation functions
+install_leveldb_ubuntu() {
+    echo "Installing LevelDB on Ubuntu..."
+    sudo apt-get update && sudo apt-get install -y libleveldb-dev
```

Comment:
This file is for leveldb, but maybe the user want to use RocksDb

Reply:
This is the tricky part, do we have any instruction of saying to install rockdb? i dont even know we need to install it.

Reply:
what about `OpenSuSE` linux? That's what i use.
```sh
zypper ref && zypper in leveldb-devel
```